src/app/public/modules/demo/demo-control-panel.component.ts

```diff
@@ -105,6 +115,10 @@ export class SkyDocsDemoControlPanelComponent implements OnDestroy, AfterContent
     this.reset.emit();
   }
 
+  public themeSettingsChange(themeSettings: SkyThemeSettings) {
```

Comment:
Code style, needs return type. I might make this method name reflect an action, such as `notifyThemeSettingsChange()`; up to you.